uppercase_vowels/script.py

```diff
@@ -0,0 +1,3 @@
+def uppercase_vowels(text: str) -> str:
```

Comment:
please move this file to script-template.py and add a new script.py with your solution code